CoEpi/ui/HealthQuiz/HealthQuizViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  HealthQuizViewController.swift
+//  CoEpi
+//
+//  Created by Dusko Ojdanic on 3/26/20.
+//  Copyright © 2020 org.coepi. All rights reserved.
+//
+
+import UIKit
+
+class HealthQuizViewController: UIViewController {
```

Comment:
Would it be possible to not touch/move the health quiz code? it'll create merge conflicts for me in https://github.com/Co-Epi/app-ios/pull/12